compiler/test/dotty/tools/dotc/reporting/ErrorMessagesTests.scala

```diff
@@ -803,4 +803,23 @@ class ErrorMessagesTests extends ErrorMessagesTest {
         assertEquals(err, ExpectedClassOrObjectDef())
       }
 
+  @Test def superCallsNotAllowedInline =
+  checkMessagesAfter("refchecks") {
+       """
+        |class A {
+        |  def foo(): Unit = ()
+        |}
+        |
+        |class B extends A {
+        |  @inline def bar(): Unit = super.foo()
```

Comment:
FYI `inline` is now a keyword in Dotty and could be used here instead - not necessary to get in though :)